src/views/AssignPickerModal.vue

```diff
@@ -190,54 +190,33 @@ export default defineComponent({
     },
     async findPickers() {
       this.isLoading = true;
-
       let inputFields = {}
       this.pickers = []
 
       if(this.queryString.length > 0) {
-        inputFields = {
-          firstName_value: this.queryString,
-          firstName_op: 'contains',
-          firstName_ic: 'Y',
-          firstName_grp: '1',
-          externalId_value: this.queryString,
-          externalId_op: 'contains',
-          externalId_ic: 'Y',
-          externalId_grp: '2',
-          lastName_value: this.queryString,
-          lastName_op: 'contains',
-          lastName_ic: 'Y',
-          lastName_grp: '3',
-          partyId_value: this.queryString,
-          partyId_op: 'contains',
-          partyId_ic: 'Y',
-          partyId_grp: '4',
-          groupName_value: this.queryString,
-          groupName_op: 'contains',
-          groupName_ic: 'Y',
-          groupName_grp: '5'
-        }
+        inputFields = `firstName:*${this.queryString}* OR externalId:*${this.queryString}* OR lastName:*${this.queryString}* OR partyId:*${this.queryString}* OR groupName:*${this.queryString}*`
+      }
+      else {
+        inputFields = `*:*`
       }
 
       const payload = {
-        inputFields: {
-          ...inputFields,
-          roleTypeIdTo: 'WAREHOUSE_PICKER'
-        },
-        viewSize: 50,
-        entityName: 'PartyRelationshipAndDetail',
-        noConditionFind: 'Y',
-        orderBy: "firstName ASC",
-        filterByDate: "Y",
-        distinct: "Y",
-        fieldList: ["firstName", "groupName", "lastName", "partyId", "externalId"]
+        "json": {
+          "params": {
+            "rows": "50",
+            "q": inputFields,
+            "fl": "firstName externalId lastName partyId groupName",
+            "sort": "firstName asc"
```

Comment:
Use edismax parser here. You need to add "defType" : "edismax" in the params. Also specify query fields by adding "qf": "firstName, lastName, groupName, partyId, externalId".